test/metabase/models/login_history_test.clj

```diff
@@ -38,7 +38,8 @@
 (deftest send-email-on-first-login-from-new-device-test
   (testing "User should get an email the first time they log in from a new device (#14313, #15603)"
     (mt/with-temp User [{user-id :id, email :email, first-name :first_name}]
-      (let [device (str (java.util.UUID/randomUUID))]
+      (let [device              (str (java.util.UUID/randomUUID))
+            original-maybe-send (var-get #'login-history/maybe-send-login-from-new-device-email)]
```

Comment:
You shouldn't need to `(var-get a-var)` here... just doing `a-var` should give you the same thing

Reply:
it's private so i can't just use the symbol `login-history/maybe-send`, I need the var reader macro `#login-history/maybe-send`. But this is the var, and when i redef it this var points at the redef'd version, so i need its value at this point, so I need var-get.

```clojure
user=> (ns foo)
nil
foo=> (defn- foo [] :original)
#'foo/foo
bar=> (ns bar)
nil
bar=> (let [original #'foo/foo]
        (with-redefs [foo/foo (fn [] :covered)]
          [(#'foo/foo) (original)]))
[:covered :covered] ;; :covered because original is a var, not the function value
bar=> (let [original (var-get #'foo/foo)]
        (with-redefs [foo/foo (fn [] :covered)]
          [(#'foo/foo) (original)]))
[:covered :original]
bar=> 
```